goworker.go

```diff
@@ -23,19 +23,20 @@ var (
 var workerSettings WorkerSettings
 
 type WorkerSettings struct {
-	QueuesString   string
-	Queues         queuesFlag
-	IntervalFloat  float64
-	Interval       intervalFlag
-	Concurrency    int
-	Connections    int
-	URI            string
-	Namespace      string
-	ExitOnComplete bool
-	IsStrict       bool
-	UseNumber      bool
-	SkipTLSVerify  bool
-	TLSCertPath    string
+	QueuesString       string
+	Queues             queuesFlag
+	IntervalFloat      float64
+	Interval           intervalFlag
+	Concurrency        int
+	Connections        int
+	URI                string
+	Namespace          string
+	ExitOnComplete     bool
+	IsStrict           bool
+	UseNumber          bool
+	SkipTLSVerify      bool
+	TLSCertPath        string
+	prefillParallelism int
```

Comment:
Is it okay to use `camelCase` here? In the README you are using PascalCase.

Reply:
Thanks for pointing, fixed it

Reply:
Glad to be able to help.